tools/Orca.py

```diff
@@ -0,0 +1,74 @@
+#idk, license or something
+# W3SLAV was x
+
+
+#this tool only works on x86 linux platforms for the time being (because Wonderful Toolchain doesn't support windows)
+import os
+
+#urllib is only for downloading Wonderful Toolchain builds
+from urllib.request import urlretrieve
+
+#this is for simple "portable" option (no path exports and all files sent to single dir)
+import argparse
+parser = argparse.ArgumentParser()
+parser.add_argument("--portable",action="store_true", help="build in portable mode")
+args = parser.parse_args()
+
+
+print("OrcaBlocks Installer v1.0\n    -W3SLAV")
+
+is_portable = False
+
+if (args.portable==True):
+    print("using portable mode")
+    print("THIS MODE IS NOT RECOMMENDED FOR NORMAL USE")
+    is_portable = True
+
+
+#pull from https://wonderful.asie.pl/bootstrap/wf-bootstrap-x86_64.tar.gz
+urlretrieve("https://wonderful.asie.pl/bootstrap/wf-bootstrap-x86_64.tar.gz", "wf-bootstrap-x86_64.tar.gz")
+
+#get current path for later
+#cwd = blocks sdk dir
+cwd = os.getcwd()
+
+if (is_portable==False):
+    #following the tutorial here:
+    os.system('mkdir /opt/wonderful')
```

Comment:
I don't really see how python helps here. All you're doing is calling `os.system()` a few times (and not checking for errors). This python script would be pretty easy to write in bash, with a `set -e` at the top to exit on any error. That way the code would be cleaner (no need for all the `os.system()` function calls) and safer (it stops if it finds something weird).